src/main/java/network/brightspots/rcv/CsvCvrReader.java

```diff
@@ -98,12 +110,26 @@ void readCastVoteRecords(List<CastVoteRecord> castVoteRecords)
           rankings.add(new Pair<>(rankAsInt, candidateId));
         }
 
+        String suppliedId = this.idColumnIndex != null
+                ? csvRecord.get(this.idColumnIndex) : null;
```

Comment:
So the lack of symmetry between these three fields is that if the supplied ID is null, it remains null instead of being turned into "no supplied ID". Is there a good justification for that? If not, would love symmetry.